Makefile

```diff
@@ -1,18 +1,9 @@
-BUILDER					= ./bin/builder
-MOCHA_PHANTOMJS = ./node_modules/.bin/mocha-phantomjs
-
-MOCHA_FLAGS = \
-	--reporter spec
-
-PHANTOMJS_FLAGS = \
-	--debug=true
-
-build:
-	$(BUILDER) > opt/test/build.js
-
-test: build
-	$(MOCHA_PHANTOMJS) $(MOCHA_FLAGS) $(PHANTOMJS_FLAGS) \
-		opt/test/index.html
-
-clean:
-	@rm opt/test/build.js
+build:
+	babel lib/index.js --out-file dist/index.js

```

Comment:
Your change does require a globally installed babel and mocha. I do not think this is an improvement because
- people may use a different babel/mocha version globally then we require
- people cannot just use `npm install --dev` to install all dependencies at once

If you do not like the syntax I discovered a nice [syntax improvement](https://github.com/bodokaiser/koa-lessie/blob/master/Makefile).

```Makefile
SHELL := /bin/bash
PATH  := node_modules/.bin:$(PATH)

test:
	@mocha --harmony --reporter spec
```